presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestOracleDistributedQueries.java

```diff
@@ -391,8 +395,17 @@ public void testInsertWithCoercion()
     }
 
     @Override
-    public void testColumnName(String columnName)
+    protected Optional<String> filterColumnNameTestData(String columnName)
     {
         // table names generated has more than 30chars, max size naming on oracle.
+        return Optional.empty();
+    }
+
+    protected JdbcSqlExecutor createJdbcSqlExecutor()
+    {
+        Properties properties = new Properties();
+        properties.setProperty("user", TEST_USER);
+        properties.setProperty("password", TEST_PASS);
+        return new JdbcSqlExecutor(oracleServer.getJdbcUrl(), properties);
```

Comment:
why not 
```
    protected SqlExecutor createJdbcSqlExecutor()
    {
        return oracleServer::execute;
    }
```

Reply:
I can follow this up in a cleanup for all JDBC tests since all JDBC TestingServers do implement SqlExecutor's `execute(sql)` but don't explicitly implement that interface.

Would be nice to be consistent and encapsulate the behavior where possible.